### PR TITLE
Creating tables, adding request and Google authentication endpoints

### PR DESCRIPTION
**Pull request by mg2239**
### Summary 
This pull request creates the tables needed for the database, adds most of the needed GET and POST endpoints, as well as adds an endpoint to validate an authentication key sent from the client.

### Notes 
Currently not sure how the back end will be deployed, so for now I just created an endpoint using Express.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1,11 +1,7 @@
-const AWS = require('aws-sdk');
 require('dotenv').config();
 
-const config = new AWS.Config({
-  // hardcode the config for now, DO NOT PUSH
```
Comment (lsizemore8) [inline config.js]: Remove this comment, since we already fixed that

File: tables.js [hunk 2]
```diff
@@ -0,0 +1,242 @@
+const AWS = require('aws-sdk');
+const config = require('./config');
+
+AWS.config.update(config);
+
+// function createLocations() {
```
Comment (lsizemore8) [inline tables.js]: Understand: Why are these functions commented out and not deleted or uncommented?

File: dummydata.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+const AWS = require('aws-sdk');
```
Comment (lsizemore8) [inline dummydata.js]: What is the purpose of this file? If it's not needed, delete it. Also, rename it to be more descriptive of the needs

File: config.js (hunk 1 above)
Comment (mg2239) [reply config.js]: On my end it is removed? I'll try re-committing the file.

**Pull request merged**